primitives/npos-elections/fuzzer/src/common.rs

```diff
@@ -39,11 +39,85 @@ pub fn to_range(x: usize, a: usize, b: usize) -> usize {
 
 pub enum ElectionType {
 	Phragmen(Option<(usize, u128)>),
-	Phragmms(Option<(usize, u128)>)
+	Phragmms(Option<(usize, u128)>),
 }
 
 pub type AccountId = u64;
 
+/// Generate a set of inputs suitable for fuzzing an election algorithm
+///
+/// Given parameters governing how many candidates and voters should exist, generates a voting
+/// scenario suitable for fuzz-testing an election algorithm.
+///
+/// The returned candidate list is sorted. This sorting property should not affect the result of the
+/// calculation.
+///
+/// The returned voters list is sorted. This enables binary searching for a particular voter by
+/// account id. This sorting property should not affect the results of the calculation.
+///
+/// Note that this does not generate balancing parameters.
```

Comment:
what does it means 'balancing parameters'?

Reply:
The `balance` param here: https://github.com/paritytech/substrate/blob/8049fcfa89bfb7375f467a3f22028fea94e7bfe8/primitives/npos-elections/src/phragmen.rs#L66-L71

Reply:
ahh okay.